src/opengl/GLWin.cc

```diff
@@ -335,6 +335,10 @@ GLWin::~GLWin() {
 // }
 
 void GLWin::mainLoop() {
+  //TODO: baseInit() has to be called twice, fix it
+  //seems stupid but called baseInit twice since shapes wouldn't render if it was after, 
+  //but transform in CAD would segfault if it wasn't before
+  baseInit();  // call grails initialization for shaders
```

Comment:
I'll test the things that get called in baseInit but I don't think this is acceptable for use in the rest of the project.

Reply:
After removing the first baseInit, it looks like its still working. Going to check in with @kaiqichee, but I think this can be removed.